### ISSUE TITLE
Make DocC Article - Changelog

### ISSUE THREAD

**Issue by mikaelacaron** [mikaelacaron/Basic-Car-Maintenance#352]
## Describe the feature you want the app to have that it currently doesn't
* Create a new article in the DocC documentation called "Changelog", and write a list of features for this version, `v1.0` about the list of features in the app
* This will be added onto as the app grows and new versions are released

## Describe alternatives you've considered
We'll add this to the SettingsView later, but this changelog (for this issue) is only for adding it to DocC

## Additional context
Format like:

```
# Changelog
Features and changes between released versions on the App Store (link coming soon)

## v1.0
* Enter unlimited vehicles
* .... etc
```

**Comment by mikaelacaron** [comment]
@Nithya-Reddy-M Yes I can assign you this issue, what is this "command?" maybe that you tried to run? let me know and I can look at enabling it for my repo

**Comment by Nithya-Reddy-M** [comment]
Hi Mikaela, I used the /assign-me command to request assignment, but I realize it may require a bot or GitHub Action to be enabled. 

Thanks

**Comment by mikaelacaron** [comment]
if you have a link for implementing this bot or how it works, feel free to open an issue about it and implement it for this repo!

I'd want to consider how it works though, before I'd implement it

**Comment by Nithya-Reddy-M** [comment]
> if you have a link for implementing this bot or how it works, feel free to open an issue about it and implement it for this repo!
> 
> I'd want to consider how it works though, before I'd implement it

 I'm still a beginner and new to GitHub bots, so I'm not sure how to implement one yet. I'd be happy to explore it and share any useful resources I come across, if that would be helpful.

**Comment by mikaelacaron** [comment]
That is okay! If it's an issue you'd like to learn and try to do, you're more than welcome to make an issue and try, otherwise, no worries!

**Comment by Nithya-Reddy-M** [comment]
Thanks! I’ll start looking into how GitHub bots work and see what might be useful for this repo.

I’ll open an issue if I find something promising.
